e2e/tests/configurables.rs

```diff
@@ -158,6 +158,73 @@ async fn contract_configurables() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
+async fn contract_manual_configurables() -> Result<()> {
+    setup_program_test!(
+        Abigen(Contract(
+            name = "MyContract",
+            project = "e2e/sway/contracts/configurables"
+        )),
+        Wallets("wallet")
+    );
+
+    let str_4: SizedAsciiString<4> = "FUEL".try_into()?;
+    let new_struct = StructWithGeneric {
+        field_1: 16u8,
+        field_2: 32,
+    };
+    let new_enum = EnumWithGeneric::VariantTwo;
+
+    let configurables = MyContractConfigurables::default()
+        .with_BOOL(false)?
+        .with_U8(7)?
+        .with_U16(15)?
+        .with_U32(31)?
+        .with_U64(63)?
+        .with_U256(U256::from(8))?
+        .with_B256(Bits256([2; 32]))?
+        .with_STR_4(str_4.clone())?
+        .with_TUPLE((7, false))?
+        .with_ARRAY([252, 253, 254])?
+        .with_STRUCT(new_struct.clone())?
+        .with_ENUM(new_enum.clone())?;
+
+    let contract_id = Contract::load_from(
+        "sway/contracts/configurables/out/release/configurables.bin",
+        LoadConfiguration::default(),
+    )?
+    .with_configurables(configurables)
```

Comment:
So that you may configure a contract you created via `Contract::regular` ie via code you had in memory. 

Previously we only applied configurables to contracts loaded via "load_from". If you wanted them applied to a contract you created from in memory bytes you would need to turn `MyConfigurables` into `Configurables` and call the method `update_constants_in` and give the binary. 

This test here tests that the "setter" exists on the contract itself and that it works.